tests/test_zenodo.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+
+from typing import Any, Dict
+
+import pytest
+import papis.zenodo
+import papis.yaml
+
+
+def _get_test_json(filename: str) -> Dict[str, Any]:
+    resources = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "resources", "zenodo"
+    )
+    filepath = os.path.join(resources, filename)
+    with open(filepath) as fd:
+        result = json.load(fd)
+
+    assert isinstance(result, dict)
+    return result
+
+
+@pytest.mark.parametrize(("zenodo_id",), [("7391177",), ("10794563",)])
+def test_zenodo_id_to_data(monkeypatch: Any, zenodo_id: str) -> None:
+    mock_data_file = "input_{}.json".format(zenodo_id)
+    expected_data_file = "output_{}.json".format(zenodo_id)
```

Comment:
Small nitpick here: most of the test resources are named `IDENTIFIER.EXT` and `IDENTIFIER_out.EXT`. These should be named the same for consistency.